cdap-proto/src/main/java/co/cask/cdap/proto/RunRecord.java

```diff
@@ -31,9 +31,13 @@
   @SerializedName("runid")
   private final String pid;
 
-  @SerializedName("start")
+  @SerializedName("starting")
```

Comment:
this should be nullable since it doesn't exist in run records from previous versions